include/fiber_backend_fftw.h

```diff
@@ -119,4 +120,4 @@ void compute_z2d_fftw( int const inbox_low[3], int const inbox_high[3],
 #endif
 
 
-#endif  //! FIBER_BACKEND_FFTW_H
```

Comment:
If contributions can be made to other libraries that have not been assigned to, e.g. FFTW or P3DFFT, please let them not be just removing an end of line?